consensus/polybft/consensus_runtime.go

```diff
@@ -118,10 +119,13 @@ type consensusRuntime struct {
 
 	// logger instance
 	logger hcf.Logger
+
+	// ctx is a context instance used for goroutines shutdown
+	ctx context.Context
```

Comment:
Could we find a solution without storing context? It's not very recommended practice.

Reply:
Is this approach any better? https://github.com/0xPolygon/polygon-edge/pull/1080/commits/221ea2f0d9e0580dfbe3b7b5df93931f802649f4

Unfortunately, I don't see it is feasible to remove it from `Polybft` state (https://github.com/0xPolygon/polygon-edge/pull/1080/files#diff-f66bb89ba6547fbe9286cab6d8318c1df7257099c5139d82b03955e2f4fc5462R68-R69).

Reply:
It is better. It might be also make sense to store done channel instead of context itself. How do you think?

Reply:
> It might be also make sense to store done channel instead of context itself. How do you think?
Not sure about it, because I need `context` instance in the tracker itself (https://github.com/0xPolygon/polygon-edge/blob/EVM-126-handle-go-routines-in-event-tracker/tracker/event_tracker.go#L70). So I need to pass in context to the `consensusRuntime`, which propagates it to `EventTracker` (abstraction around `ethgo.Tracker`). If I'm correct, that would mean that we need both `context` and `doneChannel` stored in the polybft as well as `cancelFn` callback (to signal context about cancellation). Seems that would introduce confusion. WDYT?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1080/commits/65734416be50f46c4e9bee510c17904ccdb27d9e